pkg/controlplane/instance.go

```diff
@@ -480,9 +477,9 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 				clock.RealClock{},
 				kubeClient,
 				holderIdentity,
-				int32(c.ExtraConfig.IdentityLeaseDurationSeconds),
+				3600,
```

Comment:
nit:

Inside this function create a `const` for this an re-use it in the two spots:

```go
const identityLeaseDurationSeconds = 3600
```